test/evergreen.yml

```diff
@@ -5347,7 +5356,6 @@ buildvariants:
     ENABLE_TCMALLOC: 0
     additional_env_vars: |
       export LSAN_OPTIONS="$COMMON_SAN_OPTIONS:print_suppressions=0:suppressions=$(git rev-parse --show-toplevel)/test/evergreen/asan_leaks.supp"
-      export LD_PRELOAD="/usr/lib/x86_64-linux-gnu/libeatmydata.so:$LD_PRELOAD"
```

Comment:
Technically we could keep eatmydata in places like this since they don't use TCMalloc, however I compared the runtimes of tests that no longer using eatmydata and the only test with a detectable slowdown was `fops`. 
I'm a bit surprised, but eatmydata seems to have been a no-op for us.

Reply:
My preference was to scrub `libeatmydata.so`, now given you've found it is a no-op it is now a *strong* preference. So I'd prefer to leave as-is and "do not use" list it.

Reply:
Agreed on that.
Sorry, I didn't make my intention clear. This comment was to let other reviewers know I found no negative consequences from dropping eatmydata.

Reply:
Thanks for the data @ajmorton. I'm also a bit surprised about the neglegible effect of the eatmydata lib removal. I agree with the decision you made here.